service/src/main/java/com/fecklessweasel/service/DepartmentServlet.java

```diff
@@ -0,0 +1,54 @@
+package com.fecklessweasel.service;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import com.fecklessweasel.service.datatier.SQLSource;
+import com.fecklessweasel.service.datatier.SQLInteractionInterface;
+import com.fecklessweasel.service.objectmodel.*;
+import com.fecklessweasel.service.*;
+
+@WebServlet("/servlet/department")
+public final class DepartmentServlet extends HttpServlet {
+
+    /**
```

Comment:
Function's java doc should be over the function, not the fields. Fields should also have their own Javadoc comments.